### ISSUE TITLE
Support Multiple markers per line

### ISSUE THREAD

**Issue by dummifiedme** [esm7/obsidian-map-view#35]
I tried to put three locations one after another, and it didn't seem to pick all three, but only the first.

eg.

```
[](location1), abcd [](location2) and [](location3)
```

**Comment by esm7** [comment]
Indeed, inline locations are one per line. That's by design because there are also the optional inline tags, and I didn't want the syntax to get too cumbersome. If you find that too limiting, would love to get a suggestion that wouldn't be too confusing for an average user :)

**Comment by dummifiedme** [comment]
I think since we allow only one inline tag, we can simply have a system where the locations end as soon as we put anything other than `tag:....`. I think its easy to be used in both one-per-line case or multiple-per-line case.

eg. 

 ```
[](location1), abcd [](location2) and [](location3)

**Comment by esm7** [comment]
I'll definitely do [your request](https://github.com/esm7/obsidian-map-view/issues/32) for multiple tags. But generalizing on what you wrote, as soon as another location starts in a line, the plugin can associate following tags to that latest location :sunglasses: 
Will put that into work together with the multiple tags.

**Comment by dummifiedme** [comment]
> I'll definitely do [your request](https://github.com/esm7/obsidian-map-view/issues/32) for multiple tags. But generalizing on what you wrote, as soon as another location starts in a line, the plugin can associate following tags to that latest location sunglasses Will put that into work together with the multiple tags.

Exactly what I meant :)
It will be easy enough for anyone.

**Comment by esm7** [comment]
Just released 1.1.0 which should support multiple markers as discussed. Let me know how it works for you :)

**Comment by dummifiedme** [comment]
Amazing update!!
